src/main/java/org/mule/extensions/jms/internal/connection/provider/GenericConnectionProvider.java

```diff
@@ -93,4 +122,59 @@ public JmsSupport create102bSupport() {
       }
     };
   }
+
+  protected void configureSSLContextIfNeeded() {
+    if (!trustStorePassword.isEmpty() && !trustStoreName.isEmpty()) {
+      try {
+        final SSLContext context = SSLContext.getInstance(PROTOCOL);
```

Comment:
Shouldn't it be better to take the current protocol instead of setting only SSL, because it would be a strong protocol and we could be degrading that.
What about something like this?
` final SSLContext context = SSLContext.getInstance(SSLContext.getDefault().getProtocol());`

Reply:
Thanks for the suggestion, we made the change to get the dynamic protocol